packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -363,12 +375,84 @@ class EditableText extends StatefulWidget {
   /// container will start with enough vertical space for one line and
   /// automatically grow to accomodate additional lines as they are entered.
   ///
-  /// If it is not null, the value must be greater than zero. If it is greater
-  /// than 1, it will take up enough horizontal space to accomodate that number
-  /// of lines.
+  /// If this is not null, the value must be greater than zero, and it will lock
+  /// the input to the given number of lines. Setting [minLines] as well allows
+  /// the input to grow between the indicated range.
+  ///
+  /// The full set of behaviors possible with [minLines] and [maxLines] are as
+  /// follows.
+  ///
+  /// {@tool sample}
```

Comment:
Yeah, I'm not sure if this counts as "sample code".  This looks a lot more like just usage documentation.

The sample code is generally meant to be a block of code that the user can copy/paste into their app and run to see what happens, and while it could be one line of code, this seems a little too trivial.  Each one of these will be expanded into a separate blue block in the documentation with a tab on it.

Reply:
Ah got it thanks, I'm changing it to just use tick marks.